assets/javascript/highlight.min.js

```diff
@@ -0,0 +1,3 @@
+/*! highlight.js v9.11.0 | BSD3 License | git.io/hljslicense */
```

Comment:
Is this file needed actually? All highlighting happens on the server side, right? IF we don't needed, let's remove it, avoiding any confusion.

Reply:
I think we need this. It highlight keywords, differences can be seen below.
without `highlight.js`
![Screenshot from 2021-01-28 17-35-20](https://user-images.githubusercontent.com/2117670/106136549-78cf4800-618f-11eb-87cf-30695aa22eaa.png)

with `hightlight.js`
![Screenshot from 2021-01-28 17-34-53](https://user-images.githubusercontent.com/2117670/106136644-943a5300-618f-11eb-8d6a-33ff22e10955.png)